region/maridia/inner-pink/Halfie Climb Room.json

```diff
@@ -858,8 +868,64 @@
         "canHorizontalShinespark",
         {"shinespark": {"frames": 91, "excessFrames": 4}}
       ],
+      "reusableRoomwideNotable": "Halfie Climb Enter with Shinespark from the Bottom Right",
+      "note": [
+        "Quickly shoot out the 3 shot blocks then horizontal spark breaking the speed blocks.",
+        "Wait for the Oums to roll on their own to a place where they can be climbed.",
+        "Do not stand on the Oums when they begin to attack."
+      ]
+    },
+    {
+      "link": [3, 2],
+      "name": "Halfie Climb Enter with Shinespark from the Bottom Right (Conserve Health)",
+      "notable": true,
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 125
+        }
+      },
+      "requires": [
+        "Morph",
+        "canSuitlessMaridia",
+        "canUseEnemies",
+        "canHorizontalShinespark",
+        "canShinechargeMovementComplex",
+        {"shinespark": {"frames": 70, "excessFrames": 4}}
+      ],
+      "reusableRoomwideNotable": "Halfie Climb Enter with Shinespark from the Bottom Right",
       "note": [
         "Quickly shoot out the 3 shot blocks then horizontal spark breaking the speed blocks.",
+        "Jump forward to save some health before Shinesparking",
+        "Wait for the Oums to roll on their own to a place where they can be climbed.",
+        "Do not stand on the Oums when they begin to attack."
+      ]
+    },
+    {
+      "link": [3, 2],
+      "name": "Halfie Climb Enter with Shinespark from the Bottom Right (SpeedBall)",
+      "notable": true,
+      "entranceCondition": {
+        "comeInSpeedballing": {
+          "runway": {
+            "length": 0,
+            "openEnd": 0
+          }
+        }
+      },
+      "requires": [
+        "canSpeedball",
+        "canCrossRoomJumpIntoWater",
+        "canUseEnemies",
+        "canChainTemporaryBlue",
```

Comment:
i was wondering if this could be easier with spring ball, to avoid temporary blue or at least the chain, but I'm thinking probably not.

Reply:
Springball does make it easier.  You can jump through the first wall as ball then unmorph and fall into the remaing speed blocks.
But if you have springball springball jumps are available and that is much less complicated.

Reply:
Ha, of course